db/migrate/20251113120000_add_cloud_billing_to_host_facets.rb

```diff
@@ -0,0 +1,47 @@
+class AddCloudBillingToHostFacets < ActiveRecord::Migration[6.1]
+  def up
```

Comment:
Rails also supports the [change method](https://guides.rubyonrails.org/active_record_migrations.html#using-the-change-method) which it can reverse itself, so you don't need to write both `up` and `down`.